src/components/ArhPriorWork/ArhPriorWork.jsx

```diff
@@ -0,0 +1,40 @@
+import ArhPriorWorkExperience from '../ArhPriorWorkExperience/ArhPriorWorkExperience';
+import ArhPriorWorkAwards from '../ArhPriorWorkAwards/ArhPriorWorkAwards';
+
+export default function ArhPriorWork({ isEdit, errors, register, control }) {
+  return (
+    <details className="details" open>
+      <summary className="summaru">Предыдущая работа</summary>
```

Comment:
```suggestion
      <summary className="summary">Предыдущая работа</summary>
```

я искал, где этот класс объявляется и удивился, что не импортируется файл ArhPriorWork.scss. Потом я увидел, что этот класс в другом месте объявляется. Я так понял, что он используется в нескольких компонентах. В таком случае, надо делать отдельный реактовый компонент Summary и переиспользовать его. В переиспользуемых классах легко запутаться, и вся мысль вокруг css направлена на решение именно этой проблемы, Так придумали и БЭМ и css-модули